pxr/usd/plugin/usdAbc/CMakeLists.txt

```diff
@@ -10,7 +10,6 @@ set(optionalLibs "")
 set(optionalIncludeDirs "")
 
 if (PXR_ENABLE_HDF5_SUPPORT)
-   add_definitions(-DPXR_HDF5_SUPPORT_ENABLED)
```

Comment:
Is this simply removed because it is not used at all ?

Reply:
No, I moved it to `Packages.cmake` so it's consistent with the rest of the package support definitions: https://github.com/PixarAnimationStudios/OpenUSD/pull/3576/files#diff-4af9a512717a73d6c1ef832e7e6a48b6ddab7568cb393063d48a0b8656b393c8L261